about.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+    <title>About</title>
+</head>
+
+<body>
+    <!--Header Section-->
+    <header id="headerSection" class="header-section">
+        <div class="top-bar">
+            <div class="top-bar-container">
+                <ul class="social-media-list">
+                    <li class="social-media-item">
+                        <i class="social-media-icon fa-brands fa-facebook-f"></i>
+                    </li>
+                    <li class="social-media-item">
+                        <i class="social-media-icon fa-brands fa-twitter"></i>
+                    </li>
+                </ul>
+                <ul class="login-bar">
+                    <li class="login-item">English</li>
+                    <li class="login-item">Mypage</li>
+                    <li class="login-item">Log Out</li>
+                </ul>
+            </div>
+        </div>
+        <section class="header-container">
+            <a href="index.html" class="logo-link">
```

Comment:
[OPTIONAL]
- The logo here appears much smaller than the home page's logo. Please make them a similar size .such that your navigation bar can look uniform on both pages.
### About Page
![image](https://github.com/CVILLA09/capstone-project/assets/45460192/e1174aa6-d379-4842-86b6-1bcf6b18a07e)
### Home Page
![image](https://github.com/CVILLA09/capstone-project/assets/45460192/01bba4c4-f317-4611-b2ce-d77eeb9f7740)